sale_order_line_sequence/__manifest__.py

```diff
@@ -5,20 +5,13 @@
 {
     "name": "Sale Order Line Sequence",
     "summary": "Propagates SO line sequence to invoices and stock picking.",
-    "version": "12.0.1.0.0",
-    "author": "Eficent, "
-              "Serpent CS, "
-              "Odoo Community Association (OCA)",
+    "version": "13.0.1.0.0",
+    "author": "Eficent, " "Serpent CS, " "Odoo Community Association (OCA)",
```

Comment:
I think the double double quotes are not necessary.

Reply:
@rousseldenis thank for review, Fixed it.